e2e/imgbuild/imgbuild.go

```diff
@@ -166,17 +166,13 @@ func (c imgBuildTests) nonRootBuild(t *testing.T) {
 		},
 		{
 			name:      "library sif",
-			buildSpec: "library://sylabs/tests/busybox:1.0.0",
+			buildSpec: "library://busybox:1.31.1",
 		},
 		{
 			name:      "library sif sandbox",
-			buildSpec: "library://sylabs/tests/busybox:1.0.0",
+			buildSpec: "library://busybox:1.31.1",
 			args:      []string{"--sandbox"},
 		},
-		{
-			name:      "library sif sha",
-			buildSpec: "library://sylabs/tests/busybox:sha256.8b5478b0f2962eba3982be245986eb0ea54f5164d90a65c078af5b83147009ba",
```

Comment:
This is removed, as the library interaction only has one path now, which is exercised in a `pull` test that can work on all architectures with a single `sha256.`